homeassistant/components/overkiz/icons.json

```diff
@@ -0,0 +1,15 @@
+{
+  "entity": {
+    "climate": {
+      "overkiz": {
+        "state_attributes": {
+          "preset_mode": {
+            "state": {
+              "drying": "mdi:hair-dryer"
```

Comment:
The frontend currently use [water-percent](https://pictogrammers.com/library/mdi/icon/water-percent/) for dry mode, while I understand here it is a preset (should it be a preset or HVAC mode?) maybe it worth use the same icon?
https://github.com/home-assistant/core/blob/a7922690c435c240e37cac3d11400cd7c3c38510/homeassistant/components/climate/icons.json#L25

![image](https://github.com/user-attachments/assets/62704d5e-2c71-4272-bc70-a62e0ce42865)

Reply:
In this device the 'drying' preset is used on top of the existing HVACMode, specific as preset for towel drying. I don't have all details, but in this context it did make sense to differentiate it from the dry icon of the HVACMode.

Does this make sense? Or should I chase an actual owner of this device to figure out more about this specific preset 😀.

Reply:
I think your explanation makes sense to use a different icon